ClientApp/src/api-authorization/login/login.component.ts

```diff
@@ -102,8 +103,8 @@ export class LoginComponent implements OnInit {
     });
   }
 
-  private getReturnUrl(state?: INavigationState): string {
-    const fromQuery = (this.activatedRoute.snapshot.queryParams as INavigationState).returnUrl;
+  private getReturnUrl(state?: NavigationState): string {
```

Comment:
Codacy found an issue: [Method 'getReturnUrl' has a complexity of 7. Maximum allowed is 4.](https://app.codacy.com/gh/InvestingOak/InvestingOak/pullRequest?prid=6277153)